reconcile/utils/jjb_client.py

```diff
@@ -22,13 +22,18 @@
 
 from reconcile.exceptions import FetchResourceError
 
+JJB_INI = "[jenkins]\nurl = https://JENKINS_URL"
+
 
 class JJB(object):
     """Wrapper around Jenkins Jobs"""
 
-    def __init__(self, configs, ssl_verify=True, settings=None):
+    def __init__(self, configs, ssl_verify=True,
+                 settings=None, local=False):
```

Comment:
i very much like the usage of `local` here. it's very understandable. but it's called print-only on the outside and then it's called local on the inside. why not just call in print-only?